packages/block-library/src/query/hooks.js

```diff
@@ -40,7 +40,7 @@ const queryTopInspectorControls = createHigherOrderComponent(
 
 		return (
 			<>
-				<InspectorControls>
+				<InspectorControls __experimentalGroup="info">
```

Comment:
The only reason we used a filter before was to render this at the top of the `InspectorControls` slots. With a new dedicated `slot` we can move this inside `query/edit/inspector-controls` file and remove the filter altogether.

Reply:
Thanks, much cleaner 👍 (https://github.com/WordPress/gutenberg/pull/45437/commits/81e582a0090ad8ca4234049a3b2ae4a3c47a7b17)